{
  "contest": 420,
  "handle": "yaioxy",
  "risk": "3",
  "title": "Funds Are Not Refunded in `EntityForging::forgeWithListed` Function",
  "issueId": 774,
  "issueUrl": "https://github.com/code-423n4/2024-07-traitforge-findings/issues/774",
  "originalIssueId": 513,
  "originalIssueUrl": "https://github.com/code-423n4/2024-07-traitforge-validation/issues/513"
}